samples/addons/prometheus.yaml

```diff
@@ -431,43 +457,48 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   labels:
-    component: "server"
-    app: prometheus
-    release: prometheus
-    chart: prometheus-19.6.1
-    heritage: Helm
+    app.kubernetes.io/component: server
+    app.kubernetes.io/name: prometheus
+    app.kubernetes.io/instance: prometheus
+    app.kubernetes.io/version: v2.47.0
+    helm.sh/chart: prometheus-25.2.0
+    app.kubernetes.io/managed-by: Helm
+    app.kubernetes.io/part-of: prometheus
   name: prometheus
   namespace: istio-system
 spec:
   selector:
     matchLabels:
-      component: "server"
-      app: prometheus
-      release: prometheus
+      app.kubernetes.io/component: server
+      app.kubernetes.io/name: prometheus
+      app.kubernetes.io/instance: prometheus
```

Comment:
fwiw this makes it impossible to upgrade. https://github.com/prometheus-community/helm-charts/pull/3246/files#diff-b1f5a77d2d0649cfb64d4617b41aabfa63c2eb40c8fe1501a81697770017efd6R68.

Which is fine -- this is a sample after all. Just something to keep in mind